cg/meta/orders/ticket_handler.py

```diff
@@ -151,3 +154,34 @@ def replace_empty_string_with_none(cls, obj: Any) -> Any:
                 else:
                     obj[key] = cls.replace_empty_string_with_none(item)
         return obj
+
+    def create_trimmed_new_ticket_message(
+        self, order: OrderIn, user_name: str, project: str
+    ) -> str:
+        """Creates a new order message but trims away the data type and encoding information"""
+        return self.create_new_ticket_message(order=order, user_name=user_name, project=project)[
+            len(self.MESSAGE_PREFIX) + 2 :
+        ]
+
+    def connect_to_ticket(
+        self, order: OrderIn, user_name: str, user_mail: str, project: str, ticket_number: int
+    ) -> None:
+        """Appends a new order message to the ticket selected by the customer"""
+        LOG.info("Connecting order to ticket %s", ticket_number)
+        sender_prefix, email_server_alias = user_mail.split("@")
+        json_attachment: NamedTemporaryFile = self.osticket.create_connecting_ticket_attachment(
+            content=self.replace_empty_string_with_none(obj=order.dict())
+        )
+        email_form = FormDataRequest(
+            sender_prefix=sender_prefix,
+            email_server_alias=email_server_alias,
+            request_uri=self.osticket.mail_container_uri,
```

Comment:
Why is this _request_uri_ on the lhs and _mail_container_uri_ on the rhs? would it make sense to have the same name on both sides?